torch_geometric/nn/conv/sage_conv.py

```diff
@@ -92,7 +123,21 @@ def forward(self, x: Union[Tensor, OptPairTensor], edge_index: Adj,
     def message(self, x_j: Tensor) -> Tensor:
         return x_j
 
-    def message_and_aggregate(self, adj_t: SparseTensor,
-                              x: OptPairTensor) -> Tensor:
+    def message_and_aggregate(self, adj_t: SparseTensor, x: OptPairTensor,
+                              edge_index_j, edge_index_i) -> Tensor:
+        """
+            Performs both message passing and aggregation of messages from neighbours using the aggregator_type
+        """
         adj_t = adj_t.set_value(None, layout=None)
-        return matmul(adj_t, x[0], reduce=self.aggr)
+        if self.lstm is not None:
+            x_j = x[0][edge_index_j]
+            x, mask = to_dense_batch(x_j, edge_index_i)
+            _, (rst, _) = self.lstm(x)
+            out = rst.squeeze(0)
+            return out
+
+        elif self.aggr == 'mean':
```

Comment:
```suggestion
        else:
```
should work here as well.